src/ts/constants/index.ts

```diff
@@ -0,0 +1,40 @@
+const MAX_NAME_LENGTH = 10;
+const PRICE_RULE = {
+  MIN: 100,
+  MAX: 10000,
+  UNIT: 10,
+};
+const MAX_QUANTITY = 20;
+
+const coinType: [500, 100, 50, 10] = [500, 100, 50, 10];
+
+const CASH_RULE = {
+  MIN: 1,
+  MAX: 100000,
```

Comment:
```suggestion
  MAX: 100_000,
```
큰 정수를 입력할때 언더스코어를 이용하면 가독성이 좋아요!
[지원](https://caniuse.com/mdn-javascript_grammar_numeric_separators)도 IE랑 오페라를 제외하고는 지원해요.. 😅

Reply:
아앗.. 비운의 IE는 여기에서도..🤣 아사님 덕분에 새로운걸 알게 됐네요! 감사합니다 적용해볼게요 :) 
https://github.com/woowacourse/javascript-vendingmachine/pull/12/commits/88cd39991387d9750d95bab3f29765d9f0531851